src/main/java/org/opentripplanner/model/impl/EntityById.java

```diff
@@ -4,10 +4,18 @@
 
 import java.io.Serializable;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 
-public class EntityMap<I extends Serializable, E extends IdentityBean<I>> {
+/**
+ * The purpose of this class is to provide a map from id to the corresponding entity.
+ * It is simply an index of entities.
+ *
+ * @param <I> the entity id type
+ * @param <E> the entity type
+ */
+public class EntityById<I extends Serializable, E extends IdentityBean<I>> {
```

Comment:
As far as I can see, in all usages of EntityById the first parameter is a FeedScopedId. This makes sense since this data structure always stores transit data entities indexed on their IDs. Should we just de-parameterize it so the keys are hard-wired to be FeedScopedIds?

Reply:
I think it used to be on place were the id was something else, it might be in NeTEx import it self. I will add it to the #2757 cleanup list to fix it - if it is still valid.